core/resources/text.vs

```diff
@@ -0,0 +1,81 @@
+#ifdef GL_ES
+precision mediump float;
+#endif
+
+attribute lowp float a_fsid;
+attribute vec2 a_position;
+attribute vec2 a_texCoord;
+
+uniform sampler2D u_transforms;
+uniform lowp vec2 u_tresolution;
+uniform lowp vec2 u_resolution;
+uniform mat4 u_proj;
+
+varying vec2 f_uv;
+varying float f_alpha;
+
+#define PI 3.14159
+
+#define alpha   tdata.a
+#define tx      tdata.x
+#define ty      tdata.y
+#define theta   tdata.z
+#define txp     tdataPrecision.x
+#define typ     tdataPrecision.y
+
+/*
+ * Converts (i, j) pixel coordinates to the corresponding (u, v) in
+ * texture space. The (u,v) targets the center of pixel
+ */
+vec2 ij2uv(float i, float j, float w, float h) {
```

Comment:
No code styling is necessary :) But yeah, I was wondering about it, does it really makes sense to put _ in shaders? Since putting it is usually to differentiate those variable from class members?